css/style.css

```diff
@@ -411,13 +719,178 @@
     padding-right: 35px;
   }
 
+  .data__figures {
+    font-weight: 600;
+  }
+
   .test-btn {
     padding-right: 0;
   }
 
   .testdrive {
     margin-right: 60px;
   }
+
+  .info__img {
+    height: auto;
+    width: 100%;
+    max-width: 1199px;
+  }
+
+  .wrap-information, .wrap {
+    height: 747px;
+    }
+
+  .red-first {
+    width: 400px;
+    height: 100%;
+    padding: 37px 47px 0 40px;
+  }
+
+  .description__title {
+    font-size: 25px;
+  }
+
+  .drive {
+    padding-bottom: 60px;
+  }
+
+  .img-inch {
+    width: 30px;
+    height: 47px;
+  }
+
+  .data__figure {
+    font-size: 32px;
+  }
+
+  .data__ins {
+    font-size: 19px;
+  }
+
+  .img-wifi {
+    width: 38px;
+    height: 28px;
+  }
+
+  .data__wireless-update {
+    font-size: 18px;
+  }
+
+  .description__bag {
+    padding-bottom: 90px;
+  }
+
+  .img-cap {
+    width: 30px;
+    height: 44px; 
+  }
+
+  .data__liters {
+    font-size: 18px;
+  }
+
+  .info-order {
+    margin: 0 -15px 0 12px;
+  }
+
+  .info-order__text {
+    white-space: nowrap;
+    font-size: 10px;
+  }
+
+  .info-order__btn {
+    width: 132px;
+    height: 38px;
+    font-size: 12px;
+  }
+  
+  .details {
+    width: 400px;
+    height: 748px;
+  }
+
+  .description-car {
+    padding-left: 0;
+  }
+
+  .title {
+    padding: 0px 52px 0 0;
+  }
+
+  .description__title-most {
+    font-size: 25px;
+    line-height: 28px;
+  }
+
+  .model-s {
+    font-size: 16px;
+    padding: 5px 38px 50px 0;
+  }
+
+  .rating-container {
+    padding: 0 20px 27px 0;
+  }
+
+  .rating-container__title {
+    font-size: 25px;
+    padding-bottom: 0;
+  }
+  
+  .rating__text {
+    font-size: 13px;
+    margin-top: -7px;
+  }
+
+  .side {
+    padding-right: 87px;
+  }
+
+  .rating__text-st {
+    padding-right: 38px;
+  }
+
+  .passenger {
+    padding-right: 53px;
+  }
+
+  .rating__img {
+    width: 20px;
+    height: 20px;
+  }
+
+  .description__rating {
+    padding: 0;
+  }
+
+  .data__img-sec {
+    padding: 3px 25px 60px 10px;
+    width: 56px;
+    height: 68px;
+  }
+
+  .data__stars {
+    font-size: 40px;
+    font-weight: normal;
+  }
+
+  .data__security {
+    font-size: 19px;
+    letter-spacing: 2px;
+  }
+
+  .wrap__img {
+    width: 50%;
+    height: 104px;
+    padding-left: 512px;
+    top: -55px;
+  }
+
+  .wrap__img-car {
+      max-width: 69%;
+      right: -314px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)